sampledb/src/main/java/com/rean/entity/Prod.java

```diff
@@ -0,0 +1,36 @@
+package com.rean.entity;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity (name="PROD")
+public class Prod {
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	public Long id;
```

Comment:
First we should not have public member variables without 'final' modifier.
Second if you have public member variables then why do you need setter/getter methods for them?